packages/backend-app-api/src/services/implementations/rootHttpRouter/DefaultRootHttpRouter.ts

```diff
@@ -66,6 +66,12 @@ export class DefaultRootHttpRouter implements RootHttpRouterService {
   private constructor(indexPath?: string) {
     this.#indexPath = indexPath;
     this.#router.use(this.#namedRoutes);
+
+    // Any request with a /api/ prefix will skip the index router, even if no named router matches
+    this.#router.use('/api/', (_req, _res, next) => {
+      next('router');
```

Comment:
Is this safe to do if `this.#indexPath` is false and there is no next router? :D

Reply:
Yep, since we're exiting out from the `this.#router` it doesn't matter whether we have a handler for the index router registered afterwards or not. I added a test for that case though 👍